src/test_suite/fuzz_context.py

```diff
@@ -25,3 +25,8 @@
     fixture_desc=vm_pb.SyscallFixture.DESCRIPTOR,
     effects_human_encode_fn=syscall_codec.encode_output,
 )
+
+ValidateVM = HarnessCtx(
```

Comment:
optional: you can define a custom prune function too if it makes sense

Reply:
I abstracted pruning in a separate branch, just need to make a PR and merge it in

Reply:
the vm validate effects protobuf has like two uint fields (one a boolean) so I think no pruning necessary?